src/Microsoft.SqlTools.ServiceLayer/Connection/ConnectionService.cs

```diff
@@ -1327,9 +1327,15 @@ public static SqlConnectionStringBuilder CreateConnectionStringBuilder(Connectio
                         {
                             connStringBuilder.Password = ConnectionService.PasswordPlaceholder;
                         }
-
-                        connStringBuilder.ApplicationName = "sqlops-connection-string";
-
+                        // default connection string application name to always be included unless set to false
+                        if (connStringParams.IncludeApplicationName != false)

```

Comment:
Just simplify this to 

```
if (!connStringParams.IncludeApplicationName.HasValue() || connStringParams.IncludeApplicationName.Value == true)
{
    connStringBuilder.ApplicationName = "sqlops-connection-string";
}
```

(you could also make an extension method HasFalse like we've done for HasTrue. Could be useful elsewhere)